cdm/src/main/java/ucar/nc2/dataset/CoordinateAxis.java

```diff
@@ -73,7 +73,7 @@
 
 public class CoordinateAxis extends VariableDS {
   static private org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(CoordinateAxis.class);
-  static private final int axisSizeToCache = 100 * 1000; // bytes
+  static public int axisSizeToCache = 100 * 1000; // bytes
```

Comment:
I suppose there is no way to make this package private? If not, that's ok, just checking to make sure.